configs/olcf/config.20161117.beta0.A_WCYCL1850S.ne30_oEC_ICG.titan

```diff
@@ -100,6 +97,9 @@ endYear = 140
 
 # directory where ocean observations are stored
 baseDirectory = /lustre/atlas/proj-shared/cli115/observations
+sstSubdirectory = SST
+sssSubdirectory = SSS
+mldSubdirectory = MLD
```

Comment:
I agree with your other similar comments, but this one, because we have slightly different directory structures on different machines, so it may be clearer to specify subdirectories.